frappe/public/js/website-core.bundle.js

```diff
@@ -0,0 +1,110 @@
+import "./jquery-bootstrap";
+import "./frappe/provide.js";
+import "./frappe/format.js";
+import "./frappe/utils/datatype.js";
+import "./frappe/utils/common.js";
+import "./frappe/translate.js";
+import "../../website/js/website.js";
+
+// Floating Message
+frappe.utils.icon = function (
```

Comment:
There's no changes to anything in any of these functions right? Just seems like code duplication then

Reply:
There is no changes, correct. I'll split it up the current sources accordingly, to avoid duplication.